dss/api/files.py

```diff
@@ -112,6 +116,25 @@ def get_helper(uuid: str, replica: Replica, version: str=None, token: str=None):
 def _verify_checkout(
         replica: Replica, token: typing.Optional[str], file_metadata: dict, blob_path: str,
 ) -> typing.Tuple[str, bool]:
+    try:
+        cloud_handle = Config.get_blobstore_handle(replica)
+        hca_handle = Config.get_hcablobstore_handle(replica)
+        checkout_file_key = get_dst_key(blob_path)
+        last_modified = cloud_handle.get_last_modified_date(replica.checkout_bucket, "files/{}".format(blob_path))
```

Comment:
You define `checkout_file_key` which results in `f"files/{blob_path}"` and then submit `"files/{}".format(blob_path)` to `get_last_modified_date` on the next line?

Reply:
Good catch!